### ISSUE TITLE
dependent on json-1.8.3?

### ISSUE THREAD

**Issue by DanielKehoe** [renz45/mandrill_mailer#126]
I'm attempting to install the gem on a project with Ruby 2.4.0 and Rails 5.0.1 and getting an error with ```bundle install```:

```
An error occurred while installing json (1.8.3), and Bundler cannot
continue.
Make sure that `gem install json -v '1.8.3'` succeeds before bundling.
```

**Comment by renz45** [comment]
Hi Daniel, sorry for the delayed response, holidays and all. It looks like this is an issue with the mandrill-api gem. Looking at their repo: https://bitbucket.org/mailchimp/mandrill-api-ruby/pull-requests/8/fix-json-version/diff#comment-None it looks like they have a pull request in to fix it. Once that gets merged in I can upgrade the version which should fix this issue.

**Comment by GabrielMahan** [comment]
Maybe make this gem dependent on a [github mirror of the gem?](https://github.com/killthekitten/mandrill-api-ruby/blob/master/mandrill-api.gemspec)?

**Comment by renz45** [comment]
If someone wanted to try swapping to the linked github mirror of the mandrill api gem and let me know if that fixes the issue, I'll merge it in and cut a release. I no longer have a mandrill account so it's hard for me to test big changes like swapping out the baseline api gem.

**Comment by renz45** [comment]
I just released a 1.7.0 version that fixes this 🎉 Send your thanks to @olivierlacan ❤️
